src/components/common/BreadCrumbs.vue

```diff
@@ -0,0 +1,36 @@
+<script>
+import AppButton from '@/components/common/App/AppButton.vue';
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+  components: { AppButton }
+});
+</script>
+
+<template>
+  <div class="bread-crumbs">
+    <AppButton
+      color-scheme="dark-reverse"
+      arrow="left"
+      class="bread-crumbs__button"
+      @click="this.$router.go(-1)"
+    >
+    Back</AppButton>
```

Comment:
Formatting a bit weird here, Prettier sometimes puts end tag with content on the same line. Also no indentation for `Back` text.

Reply:
Yeah, something went wrong here. Shall it look like this? 

> 
>
      Back
    </AppButton>

Reply:
Yeah, this is optimal